src/screens/signUp/validator/rules.js

```diff
@@ -0,0 +1,28 @@
+export const validationRules = [
+  {
+    field: "firstName",
```

Comment:
Form Fields/Inputs are Required this error is missing for all Input
E.g. First Name is Required. Also, the minimum length of the first name is 3.
match all validations with the backend